### PR TITLE
Launcher: Allow setting a distinct stderr redirect

### PR DESCRIPTION
**Pull request by s1341**

### CODE REVIEW

File: libafl/src/bolts/launcher.rs [hunk 1]
```diff
@@ -176,7 +185,11 @@ where
                         if !debug_output {
                             if let Some(file) = stdout_file {
                                 dup2(file.as_raw_fd(), libc::STDOUT_FILENO)?;
-                                dup2(file.as_raw_fd(), libc::STDERR_FILENO)?;
+                                if let Some(stderr) = stderr_file {
+                                    dup2(stderr.as_raw_fd(), libc::STDERR_FILENO)?;
+                                } else {
+                                    dup2(file.as_raw_fd(), libc::STDERR_FILENO)?;
```
Comment (domenukk) [inline libafl/src/bolts/launcher.rs:191]: This should probably also happen when `debug_output` is set(?)

Comment (s1341) [reply libafl/src/bolts/launcher.rs:191]: What do you mean?

Comment (domenukk) [reply libafl/src/bolts/launcher.rs:191]: If `debug_output` is `true`, the redirect won't happen, not sure if that's what we want?

Comment (s1341) [reply libafl/src/bolts/launcher.rs:191]: if `debug_output` is true, we're supposed to ignore the stdout/stderr redirections and just let things go to original stdout/stderr, no?

**Pull request merged**